src/components/one-on-one.js

```diff
@@ -90,10 +90,10 @@ function OneOnOne() {
                     <div className={"section-content content"}>
                         <div className={"section-copy"}>
                             <h1 className={"section-h1"}>1:1</h1>
-                            <p>
-                                Every end of the day,<br></br>random 1:1’s are matched and sent to you via email!<br></br>(All 1:1 are matched at 9:00AM KST)
-                            </p>
-                            <MyButton></MyButton>
+                            <p style={{'marginTop': '1.5em', 'marginBottom': '0.5em'}}>Every end of the day,</p>
```

Comment:
How about staying away from inline css?
May be hard to debug in the future.

Reply:
I don't know why these codes are there. I think some cat sneaked in my home and write these.